test/integration/pipeline/tx_pipeline_integration_test.cpp

```diff
@@ -142,24 +142,26 @@ TEST_F(TxPipelineIntegrationTest, GetTransactionsTest) {
   shared_model::crypto::CryptoModelSigner<> signer(keypair_);
   auto given_transaction = shared_model::proto::from_old(given_tx);
   signer.sign(given_transaction);
-  given_tx = *std::unique_ptr<iroha::model::Transaction>(given_transaction.makeOldModel());
+  given_tx = *std::unique_ptr<iroha::model::Transaction>(
```

Comment:
Pls abort this changes, because this code will be removed after merge https://github.com/hyperledger/iroha/pull/1151.